reactivesocket-tck-drivers/run.sh

```diff
@@ -1,3 +1,7 @@
 #!/bin/bash
 
-java -cp build/libs/reactivesocket-tck-drivers-0.2.2-SNAPSHOT.jar io.reactivesocket.tckdrivers.main.Main "$@"
+LATEST_VERSION=$(ls build/libs/ | sort -r | head -1)
```

Comment:
This could be perfectly safe, but looks brittle given the libs directory isn't just one file for a clean build.  I would suggest either tightening up the ls command to generally only find one file, or look at a command like this

http://stackoverflow.com/questions/24936781/gradle-plugin-project-version-number